docs/contributing/docs-contributions.md

```diff
@@ -190,7 +190,7 @@ Sometimes it makes sense to move or rename a file as part of docs restructuring
 
 ```yaml:title=www/redirects.yaml
 - fromPath: /docs/source-plugin-tutorial/
-  toPath: /docs/pixabay-source-plugin-tutorial/
+  toPath: /docs/tutorial/pixabay-source-plugin-tutorial.md
```

Comment:
The "to" path won't include the file extension, so you should leave `.md` off here.
```suggestion
  toPath: /docs/tutorial/pixabay-source-plugin-tutorial/
```